src/pages/keyblocks/[id].vue

```diff
@@ -14,7 +14,7 @@
     v-if="keyblockDetails"
     :keyblock-details="keyblockDetails"/>
 
-  <app-tabs v-if="!keyblockDetails.notExistent">
+  <app-tabs v-if="keyblockExists">
```

Comment:
I think there should be `is` prefix as a common programming convention we are also using
```suggestion
  <app-tabs v-if="isKeyblockExistent">
```

Reply:
Done